src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/client/JudicialApiClient.java

```diff
@@ -92,5 +99,24 @@ public RequestSpecification getMultipleAuthHeaders(String userToken) {
                 .header(AUTHORIZATION_HEADER, "Bearer " + userToken);
     }
 
+    public Object fetchUserProfiles(UserRequest userRequest, int pageSize, int pageNumber, HttpStatus expectedStatus,
+                                    String role) {
+        Response fetchResponse = getMultipleAuthHeadersInternal(role)
+                .body(userRequest).log().body(true)
+                .post("/refdata/judicial/users/fetch?page_size=" + pageSize + "&page_number=" + pageNumber)
```

Comment:
You may declare the URI - `/refdata/judicial/users/fetch?page_size=%s&page_number=%s` as a constant and use `String.format` to replace these placeholders with necessary values.

Reply:
addressed here dc7b300